docs/source/conf.py

```diff
@@ -64,6 +64,7 @@
 # Setup Sphinx
 def setup(sphinx):
     lexers['rust'] = RustLexer()
+    lexers['rust,ignore'] = RustLexer()
     lexers['toml'] = TOMLLexer()
```

Comment:
:memo: I wasn't aware that there is such a place where we can configure how \`\`\`\<lang\> is parsed.

Reply:
Me neither. In general I don't know much abut how Scylla documentation building works.